packages/aragon-cli/src/commands/start.js

```diff
@@ -10,6 +10,7 @@ const TaskList = require('listr')
 const pkg = require('../../package.json')
 const { installDeps } = require('../util')
 
+const DEFAULT_CLIENT_REPO = pkg.aragon.clientRepo
 const DEFAULT_CLIENT_VERSION = pkg.aragon.clientVersion
 const DEFAULT_CLIENT_PORT = pkg.aragon.clientPort
 
```

Comment:
Just a small fix here: `exports.command = 'start [client-version]'` should be `exports.command = 'start [client-repo] [client-version]'`